sdk/tests/client/input_selection/basic_outputs.rs

```diff
@@ -1123,44 +1123,45 @@ fn two_inputs_remainder_3() {
     });
 }
 
-#[test]
-fn another_input_required_to_cover_remainder_rent() {
-    let protocol_parameters = protocol_parameters();
-
-    let inputs = build_inputs([
-        Basic(500_000, BECH32_ADDRESS_ED25519_0, None, None, None, None, None, None),
-        Basic(600_000, BECH32_ADDRESS_ED25519_0, None, None, None, None, None, None),
-        Basic(700_000, BECH32_ADDRESS_ED25519_0, None, None, None, None, None, None),
-    ]);
-    let outputs = build_outputs([Basic(
-        1_000_000,
-        BECH32_ADDRESS_ED25519_0,
-        None,
-        None,
-        None,
-        None,
-        None,
-        None,
-    )]);
+// TODO: re-enabled when rent is figured out
```

Comment:
What's still missing?

Reply:
Just don't really know how to write this test without a point of reference, I guess